module-2/backend/week_one.md

```diff
@@ -5,30 +5,114 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+The five common verbs: 
+  ** 1. Get ** Get's purpose is to receive information.
+  ** 2. Post ** Post's purpose is to send information.
+  ** 3. Put ** Put's purpose is to update information.
+  ** 4. Delete ** Delete's purpose is to delete information.
+  ** 5. Head ** Head returns just the header, which I imagine is useful for something I haven't encountered yet.

```

Comment:
Ha! Yup. W were thinking of PATCH for that fifth one, but it's basically synonymous with PUT.